phone_verify/backends/base.py

```diff
@@ -51,18 +51,15 @@ def generate_session_token(cls, phone_number):
         data = {"phone_number": phone_number, "nonce": random.random()}
         encoded_str = jwt.encode(data, django_settings.SECRET_KEY)
         if type(encoded_str) is bytes:
-            # jwt v2.0.0 or greater: `jwt.encode()` no longer return <class 'bytes'> 
+            # jwt v2.0.0 or greater: `jwt.encode()` no longer return <class 'bytes'>
             # instead it return <class 'str'> and that's why `jwt.encode().decode()` fails
             # To support jwt v1.7.1 or greater: if `jwt.encode()` return <class 'bytes'>
             # convert it to <class 'str'>
             # Details: https://github.com/jpadilla/pyjwt
```

Comment:
If you've the link to the issue, that this change was made in, that would be helpful.

Reply:
> If you've the link to the issue, that this change was made in, that would be helpful.

Sorry for late. I think you wanted to know about PyJWT issue.

At first, I found it from the documentation [https://pyjwt.readthedocs.io/en/stable/changelog.html](https://pyjwt.readthedocs.io/en/stable/changelog.html).

PyJWT Github issue: [jwt.encode returns str instead of bytes in 2.0.0a1 #529](https://github.com/jpadilla/pyjwt/issues/529)

And PyJWT Github PR: 
-  [Return tokens as string not bytes #513](https://github.com/jpadilla/pyjwt/pull/513)
-  [Update return type for jwt.encode #521](https://github.com/jpadilla/pyjwt/pull/521)
- [Enforce .encode() return type using mypy #551](https://github.com/jpadilla/pyjwt/pull/551)

Reply:
Thanks for these. I'll modify the comment in the changes to reflect these and merge when the other PR on tox and other dependency upgrade is ready.

Reply:
Thanks! 😊